src/software/jetson_nano/services/motor.h

```diff
@@ -251,6 +308,11 @@ class MotorService
      */
     bool hasMotorReset(uint8_t motor);
 
+    // Wheel constants
+    static constexpr double MECHANICAL_MPS_PER_ELECTRICAL_RPM = 0.000111;
+    static constexpr double ELECTRICAL_RPM_PER_MECHANICAL_MPS =
+        1 / MECHANICAL_MPS_PER_ELECTRICAL_RPM;
```

Comment:
should the todo comment `// TODO (#2720): compute from robot constants (this was computed by hand and is accurate)` be moved here as well?